galaxyui/proxy.conf.js

```diff
@@ -20,7 +20,7 @@ const PROXY_CONFIG = {
         'secure': false
     },
     "/download": {
-        'target': "http://localhost:8080",
```

Comment:
Was this port change intentional? it looks like it would break other dev's setups if they used `8080`

Reply:
Oh ok, the `localhost:8080` looked like it was a default for local dev stuff. My mistake!